layouts/shortcodes/pipelines/latest-version.html

```diff
@@ -1 +1 @@
-1.8.3
+1.8.4
```

Comment:
NIT: KFP version here is for `KFP standalone deployment`, while `gke distribution` is for `full fledged kubeflow deployment`. So it will be easier if we separate this PR into two PRs:
1. This pipelines version file.
2. The rest of files under `gke/` path in this PR.

But it is only for reference, we don't need to take action to split this PR here.

Reply:
SGTM. Thanks!